zone/client.h

```diff
@@ -350,7 +351,7 @@ class Client : public Mob
 	void SendTraderPacket(Client* trader, uint32 Unknown72 = 51);
 	void SendBuyerPacket(Client* Buyer);
 	void SendBuyerToBarterWindow(Client* buyer, uint32 action);
-	GetItems_Struct* GetTraderItems();
+	GetItems2_Struct* GetTraderItems();
```

Comment:
Was this temporary? I'd prefer to not use GetThing2 GetThing3 and just have GetThing